tools/applicationinsights-web-snippet/src/snippet.ts

```diff
@@ -206,12 +207,24 @@ declare var cfg:ISnippetConfig;
                 });
                 // let message = "Load Version 2 SDK instead to support IE"; // where to report this error?
             }
+            if (appInsights.cr){
+                for (var i = 0; i < domains.length; i++){
+                    if (targetSrc.indexOf(domains[i])){
+                        domainRetryIndex = 0;
+                        break;
+                    }
+             }
             const _handleError = (evt?: any) => {
-                loadFailed = true;
                 appInsights.queue = []; // Clear the queue
                 if (!handled) {
-                    handled = true;
-                    _reportFailure(targetSrc);
+                    // start retry
+                    if (domainRetryIndex > 0 && domainRetryIndex < domains.length){
```

Comment:
`domainRetryIndex >= 0` perhaps?

But we also want to not retry `js.monitor.azure.com` if that is what they had. Likewise, if they used `az416426...` we don' really want to try that again...

Perhaps, we split this into a retry count and "startIndex" (matching the first domain) and then the "next" script is 
`nextIdx = (startIndex + domainRetryCount) % 6` (where 6 === `domains.length`)